UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -197,6 +216,11 @@ public bool HasAccess(Access access)
 		return accessSyncList.Contains((int) access);
 	}
 
+	public string GetJobTitle()
+	{
+		return jobTitle.IsNullOrEmpty() ? jobType.ToString() : jobTitle;
```

Comment:
maybe it could get JobTitle from the job SO's by comparing the JobType

Reply:
AllowedOccupationsSingleton should have a list of Occupations which is the thing I'm talking about